app/api/items/create_dependency.go

```diff
@@ -0,0 +1,136 @@
+package items
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/render"
+	"github.com/go-sql-driver/mysql"
+
+	"github.com/France-ioi/AlgoreaBackend/app/database"
+	"github.com/France-ioi/AlgoreaBackend/app/formdata"
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// swagger:model
+type itemDependencyCreateRequest struct {
+	// minimum: 0
+	// maximum: 100
+	Score int32 `json:"score" validate:"min=0,max=100"`
+	// required: true
+	GrantContentView bool `json:"grant_content_view"`
+}
+
+// swagger:operation POST /items/{dependent_item_id}/prerequisites/{prerequisite_item_id} items itemDependencyCreate
+// ---
+// summary: Create an item dependency
+// description: >
+//
+//   Creates an item dependency with parameters from the input data without any effect to access rights.
+//
+//   The user should have
+//
+//     * `can_view` >= 'info' on the `{prerequisite_item_id}` item,
+//     * `can_edit` >= 'all' on the `{dependent_item_id}` item,
+//     * if `grant_content_view` = true, the user should also have `can_grant_view` >= 'content'
+//       on the `{dependent_item_id}` item,
+//
```

Comment:
Probably it should be written what happen if the score is not given.

Reply:
fixed